py_workers/wiki2itl.py

```diff
@@ -13,7 +13,7 @@
 
 import argparse
 
-ja_patterns = [r'\[\[:(?:ファイル|画像):(.*?)\|',r'\[\[(?:ファイル|画像):(.*?)\|'] 
+ja_patterns = [r'\[\[:(?:ファイル|画像):(.*?)\|',r'\[\[(?:ファイル|画像):(.*?)\|', r'\[\[:(?:ファイル|画像):(.*?)\]\]'] 
```

Comment:
Only one of these patterns is supposed to be used, which is problematic. So I think they should be combined into one.

Also, "]]" should be recognised by a regular expression. This saves the hassle of having to exclude trailing "thumb" etc. later.

Reply:
Make sense, could you pls make some change on the "add_ja_pattern" branch for that ?

Reply:
Changed, please confirm. If it is not good, please change it back.